Makefile

```diff
@@ -100,14 +100,34 @@ clean:
 clean-cache:
 	rm -rf ${CACHEDIR}
 
-${CACHEDIR}/.libraries_token.txt: cache
+${CACHEDIR}/.guarded-repo.token: cache
+	tmpf=$(shell mktemp); \
+	chainctl auth octo-sts --identity=guarded-package-repos --scope=chainguard-dev > $${tmpf}; \
```

Comment:
I think the `;` would ignore a non-zero exitcode from `chainctl`. That would result in an empty `$${tmpf}` that would then get used as the token file. Maybe use `&&` instead?

Reply:
I like that approach better, will make the adjustment for this and library tokens